core/src/main/java/nl/nn/adapterframework/webcontrol/api/TransactionalStorage.java

```diff
@@ -446,14 +444,14 @@ public Response browsePipeMessages(
 		return Response.status(Response.Status.OK).entity(getMessages(storage, filter)).build();
 	}
 
-	private String[] getMessages(MultipartFormDataInput input) {
+	private String[] getMessages(MultipartBody inputDataMap) {
 		try {
-			Map<String, List<InputPart>> inputDataMap = input.getFormDataMap();
-			if(inputDataMap.get("messageIds") != null) {
-				String messageIds = inputDataMap.get("messageIds").get(0).getBodyAsString();
+			
+			if(inputDataMap.getAttachmentObject("messageIds", String.class) != null) {
```

Comment:
Same resolveStringFromMap applies here

Reply:
do you want to remove the "if"...? because resolveStringFromMap never return null.